import-service/serverless.ts

```diff
@@ -0,0 +1,81 @@
+import type { Serverless } from 'serverless/aws';
+
+const serverlessConfiguration: Serverless = {
+  service: {
+    name: 'import-service-buck'
+  },
+  frameworkVersion: '2',
+  custom: {
+    webpack: {
+      webpackConfig: './webpack.config.js',
+      includeModules: true
+    }
+  },
+  plugins: [
+    'serverless-webpack',
+    'serverless-dotenv-plugin'
+  ],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs12.x',
+    region: 'eu-west-1',
+    stage: 'dev',
+    iamRoleStatements: [
+      {
+        Effect: "Allow",
+        Action: ["s3:ListBucket"],
+        Resource: ["arn:aws:s3:::import-service-buck"],
```

Comment:
It would be better off using a constant for the bucket name in order to decrease possibility of error appearance.